frontend/packages/ceph-storage-plugin/src/components/ocs-install/node-list.tsx

```diff
@@ -266,16 +267,22 @@ const CustomNodeTable: React.FC<CustomNodeTableProps> = ({
   };
 
   const makeLabelNodesRequest = (selectedNodes: NodeKind[]): Promise<NodeKind>[] => {
-    return selectedNodes.map((node: NodeKind) => {
-      const patch = [
-        {
-          op: 'add',
-          path: '/metadata/labels/cluster.ocs.openshift.io~1openshift-storage',
-          value: '',
-        },
-      ];
-      return k8sPatch(NodeModel, node, patch);
+    const promises = [];
```

Comment:
@parthdhanjal  makeLabelNodesRequest definition should be like this -

```
const makeLabelNodesRequest = (selectedNodes: NodeKind[]): Promise<NodeKind>[] => {
    return selectedNodes.map((node: NodeKind) => {
      const nodeLabel = isLabelPresent(node, ocsLabel);
      if (!nodeLabel) {
        const patch = [
          {
            op: 'add',
            path: '/metadata/labels/cluster.ocs.openshift.io~1openshift-storage',
            value: '',
          },
        ];
        return k8sPatch(NodeModel, node, patch);
      }
    });
  };
```
No need to create a promise array here and pass the ocs label explicitly so that `isLabelPresent` function should be generic.